docs/articles/expensify-classic/spending-insights/Export-Expenses-And-Reports.md

```diff
@@ -127,7 +127,7 @@ Enter any of the following formulas into the Formula field for each column. Be s
 | Expenses Count | Is the number of total expenses on the report of this specific expense.|
 | {report:expensescount} | Would output 10 assuming that there were 10 expenses on the given report for this expense.|
 | Workspace Name | Is the name of the workspace applied to the report.|
-| {report:policyname} | Would output Sales assuming that the given report was under a workspace named Sales.|
+| {report:workspacename} or {report:policyname} | Would output Sales assuming that the given report was under a workspace named Sales.|
```

Comment:
Can we please use `{report:workspaceName}` for the main example and then add another row below with the formula `{report:policyName}` and a description "Same as the workspace name. Policy is an older term for workspace"?

I think using camelCase makes it a little more clear and it will work since parts are always lowercased before comparison.